setup.cfg

```diff
@@ -168,6 +168,7 @@ ignore_imports =
     #  -> openedx.features.enterprise_support.utils
     openedx.features.enterprise_support.utils -> lms.djangoapps.branding.api
     cms.djangoapps.contentstore.rest_api.v1.views.settings -> lms.djangoapps.certificates.api
+    lms.envs.common -> cms.lib.xblock.tagging.tagged_block_mixin
```

Comment:
See my other comment - I'd prefer not to add an exception here.

Reply:
Removed 👍